### PR TITLE
change name of "tourist table" to "folding table"

### PR DESCRIPTION
**Pull request by catern**
#### Summary
Content "Change name of tourist table to folding table"

#### Purpose of change

"Tourist table" as a name for a small metal folding table appears to
be unique to CDDA.  Indeed, CDDA is the first Google result for
"tourist table".  Personally, I have to remember every time what a
"tourist table" is supposed to be, so using a more common name would
be better.

#### Describe the solution

"Folding table" seems to be the most common phrase for this kind of
table.

#### Describe alternatives you've considered

"Travel table" or "camping table" are also candidates, and there are
other terms listed on https://en.wikipedia.org/wiki/Folding_table but
"folding table" is nicely generic, and doesn't state exactly what kind
of table it is, which is nice because we currently have lots of usage
of these tables both at campsites and inside residential buildings.
Which makes sense, because folding tables are quite common.

#### Testing

String-only change

### CODE REVIEW

**Comment by Maleclypse** [comment]
String only change is preferred.   Cuts down on the churn and also you don't have a migration in here from the old "id" to the new "id" which means this change would break every save file currently in existence.

**Comment by catern** [comment]
Sure, updated to be string-only for real

File: src/map_extras.cpp [hunk 1]
```diff
@@ -1448,7 +1448,7 @@ static bool mx_minefield( map &, const tripoint &abs_sub )
         m.spawn_item( { 16, 18, abs_sub.z }, itype_landmine, rng( 0, 5 ) );
         m.spawn_item( { 16, 19, abs_sub.z }, itype_landmine, rng( 0, 5 ) );
 
-        // Set some resting place with fire ring, camp chairs, tourist table and benches
+        // Set some resting place with fire ring, camp chairs, folding table and benches
```
Comment (kevingranade) [inline src/map_extras.cpp:1451]:
```suggestion
        // Set some resting place with fire ring, camp chairs, folding table and benches.
```

File: src/map_extras.cpp [hunk 2]
```diff
@@ -1448,7 +1448,7 @@ static bool mx_minefield( map &, const tripoint &abs_sub )
         m.spawn_item( { 16, 18, abs_sub.z }, itype_landmine, rng( 0, 5 ) );
         m.spawn_item( { 16, 19, abs_sub.z }, itype_landmine, rng( 0, 5 ) );
 
-        // Set some resting place with fire ring, camp chairs, tourist table and benches
+        // Set some resting place with fire ring, camp chairs, folding table and benches.
```
Comment (actual-nh) [inline src/map_extras.cpp:1451]:
```suggestion
        // Set some resting place with fire ring, camp chairs, folding table, and benches.
```

File: data/json/items/tool/deployable.json [hunk 3]
```diff
@@ -231,8 +231,8 @@
   {
     "id": "tourist_table",
     "type": "TOOL",
-    "name": { "str": "tourist table" },
-    "description": "Metal tourist table for off-road trips.  It is folded for easy transportation.",
+    "name": { "str": "folding table" },
+    "description": "Metal folding table for off-road trips.  It is folded for easy transportation.",
```
Comment (actual-nh) [inline data/json/items/tool/deployable.json:235]:
```suggestion
    "description": "Metal folding table for off-road trips.  It is currently folded for easy transportation.",
```

**Review by actual-nh — COMMENTED** [review]
Thanks!

**Pull request merged**